fulp_modules/jobs/deputy/code/deputy.dm

```diff
@@ -178,7 +178,10 @@ GLOBAL_LIST_INIT(available_deputy_depts, sortList(list(SEC_DEPT_ENGINEERING, SEC
 /// Used for Science Deputies
 /obj/item/reagent_containers/hypospray/medipen/mutadone
 	name = "mutadone medipen"
-	desc = "Hulked lings in the RD office? Space adapt traitors bombing the Armory? You know what to do! Comes with 2 uses."
```

Comment:
I'd like to retain these two uses. Either through doubling the volume or making a second pen.